react_practice/react-apps/waitlist-page-v1/src/CTAForm.js

```diff
@@ -13,12 +14,6 @@ function CTAForm() {
     e.preventDefault();
   };
 
-  const handleKeyPress = (e) => {
```

Comment:
Turns out you didn't need this at all 😄. I see that you're passing an `onSubmit` function to the `form` on [line 19](https://github.com/tyreljackson-L904/React_Dev/pull/1/files#diff-de2e24014e2a0af0d54a710cef04a6afac3d53dd49eeda78a9161121f8c8fe93R19). What's great about this is that the `enter` key on a form will generally try and submit the form for you. I found that typing something in the input, then hitting `enter` showed me the alert as I would expect to see.

Where this flow was broken was clicking the button. I've added some comments below to explain why this was broken.